### PR TITLE
[ops] WebApp: Fix team-specific alerts

### PR DESCRIPTION
**Pull request by geropl**
## Description
Fixes very noisy alerts introduced with https://github.com/gitpod-io/gitpod/pull/12514, mostly because I learned that `rate(gauge)` does not work, especially not across different implementations (prometheus vs VictoriaMetrics).

## Related Issue(s)
Context: https://github.com/gitpod-io/gitpod/issues/10722

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: operations/observability/mixins/meta/rules/components/server/alerts.libsonnet [hunk 1]
```diff
@@ -150,8 +134,9 @@
           },
           {
             alert: 'WebAppServicesCrashlooping',
-            expr: 'sum(rate(container_cpu_usage_seconds_total{container!="POD", node=~".*", pod=~"(content-service|dashboard|db|db-sync|messagebus|payment-endpoint|proxy|server|ws-manager-bridge|usage)-.*"}[5m])) by (pod, node) > 0.80',
-            'for': '15m',
+            // Reasoning: alert if any pod is restarting more than 3 times / 5 minutes.
+            expr: 'rate(kube_pod_container_status_restarts_total{container!="POD", node=~".*", pod=~"(content-service|dashboard|db|db-sync|messagebus|payment-endpoint|proxy|server|ws-manager-bridge|usage)-.*"}[15m]) * 60 * 3 > 0',
```
Comment (ArthurSens) [inline operations/observability/mixins/meta/rules/components/server/alerts.libsonnet:138]:
```suggestion
            expr: 'increase(kube_pod_container_status_restarts_total{container!="POD", node=~".*", pod=~"(content-service|dashboard|db|db-sync|messagebus|payment-endpoint|proxy|server|ws-manager-bridge|usage)-.*"}[5m]) > 3,
```

If you just want to measure raw increase in 5m, the `increase()` can help simplify the query 🙂

Comment (ArthurSens) [reply operations/observability/mixins/meta/rules/components/server/alerts.libsonnet:138]: The filter `node=~".*"` could also be removed, right? It's not actually filtering anything, just making the query heavier to the backend 😬

**Review by easyCZ — APPROVED** [review]
Tip for making the reviewer's life easier:

* 1 alert change per PR
* Link to grafana with the query and/or screenshot it against a previous incident/threshold

/hold in case you want to make any other changes based on other feedback

File: operations/observability/mixins/meta/rules/components/server/alerts.libsonnet (hunk 1 above)
Comment (geropl) [reply operations/observability/mixins/meta/rules/components/server/alerts.libsonnet]: Done.

**Comment by geropl** [comment]
/unhold

Already addressed comments,

**Pull request merged**